demo/lightning/checkpoint/multinode/README.md

```diff
@@ -0,0 +1,40 @@
+# Checkpoint Demo for PyTorch Lightning
+
+The code in this folder provides a training demo for multi node checkpointing with PyTorch Lightning. This demo is under development.
+
+## Limitations
+* Custom Strategy needs to be implemented and passed to the trainer.
+* `requirements.txt` includes gcsfs because even though it is not used for checkpointing, PyTorch Lightning's default logger also writes to the root directory where checkpoints are saved.
+
+## Running locally
+
+1. Set the environment variables required to run the demo. These include:
+  * `PROJECT`: The GCP project you are using
+  * `CKPT_DIR_PATH`: The full path of the directory in which to save checkpoints, in the format `gs://<bucket>/<directory>/`
+2. Set the optional environment variables, if desired:
+  * `NUM_LAYERS`: The number of layers in the model, which affects the size of the model and therefore the size of the checkpoints
+  * `ACCELERATOR`: Set to `gpu` if running on a GPU, or `cpu` if running on a CPU (default)
+    * If running on a GPU, you also must set `PJRT_DEVICE` to `CUDA`.
+  * `TRAIN_STRATEGY`: This demo requires custom strategy which should override DDP Strategy or FSDP Strategy. FSDP strategy will require actual GPU. 
```

Comment:
What are the valid values for this environment variable?

Reply:
Yes changed it to CPU although It just doesnt feel right that, setting it to CUDA is not an issue for single node but errors out on multi node.

Reply:
I mentioned this above, but hyperlinking to the strategy page would probably clarify this.

Reply:
Added a note saying custom strategy needs to extend StrategyClass or one of the class which extends strategy.